app/uk/gov/hmrc/individualincomedesstub/package.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualincomedesstub
+
+import java.net.URLDecoder
+
+import play.api.libs.json.Json
+import play.api.mvc.PathBindable
+import uk.gov.hmrc.domain.Nino
+import uk.gov.hmrc.individualincomedesstub.domain.EmployerReference
+import uk.gov.hmrc.individualincomedesstub.domain.ErrorInvalidPathParameter
+import uk.gov.hmrc.individualincomedesstub.domain.JsonFormatters.errorInvalidRequestParameterFormat
+
+package object Binders {
```

Comment:
In binders written in `individuals-api` we ended up:
1 - writing unit tests for them
2 - refactoring them from anonymous to concrete classes
Given the above it might be a good idea to do that here too for a `NinoPathBinder` and a `EmployerReferencePathBinder`.